phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -577,6 +582,10 @@ private void rebuildIndexes(Connection conn, Configuration conf, ArrayList<Strin
         if (!upgrade) {
             return;
         }
+        if (!rebuild) {
+            return;
+        }
+
```

Comment:
Do you want to log what type of options provided to IndexUpgradeTool for debugging purposes?

Reply:
We are already logging the options passed to IndexUpgradeTool https://github.com/apache/phoenix/blob/master/phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java#L186 and IndexTool https://github.com/apache/phoenix/blob/master/phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java#L688